data/css/modules/card/_media.scss

```diff
@@ -2,23 +2,40 @@
     background-color: white;
 
     &__caption {
-        font-size: 20px;
+        font-size: 16px;
         font-family: $body-font;
-        color: #666666;
-        padding: 20px 20px 0px 20px;
+        color: transparentize(black, 1 - 0.60);
     }
 
     &__attributionText {
         font-weight: bold;
-        font-size: 16px;
-        font-family: $display-font;
+        font-size: 14px;
+        font-family: $support-font;
         color: $accent-light-color;
-        padding: 10px 20px 20px 20px;
 
         &:hover {
             color: $accent-dark-color;
         }
     }
+
+    &__caption,
+    &__attributionText {
+        padding: 25px;
+    }
+}
+
+// When both the caption & attribution are present in the lightbox,
+// edit padding to accommodate both
+.CardMedia--caption.CardMedia--attribution {
+    .CardMedia {
```

Comment:
You described exactly how I thought it _should_ work, and you are also right that it _does_ work that way, here's the CSS that it compiles down to:
```css
.CardMedia--caption.CardMedia--attribution .CardMedia__caption {
  padding-bottom: 7px; }

.CardMedia--caption.CardMedia--attribution .CardMedia__attributionText {
  padding-top: 7px; }
```
I didn't understand why until I considered how I would have written it. I got confused by the extra level of SCSS nesting looking like it should correspond to an extra level of nesting in the widget tree. Anyway, go ahead with this.